pallene/coder.lua

```diff
@@ -479,13 +479,26 @@ function Coder:pallene_entry_point_definition(f_id)
     }))
 end
 
-function Coder:call_pallene_function(dsts, has_upvalue, f_id, base, xs)
+function Coder:call_pallene_function(dsts, f_id, base, xs, cclosure)
```

Comment:
What do you think about reordering the arguments of `call_pallene_function` so it's more similar to the order of the entry point? Now that `cclosure` is no longer optional, it doesn't have to be the last argument.

Reply:
Right. That's a good point.